app/actions/remote/user.ts

```diff
@@ -332,31 +337,44 @@ export const fetchMissingProfilesByUsernames = async (serverUrl: string, usernam
     }
 };
 
-export const updateAllUsersSinceLastDisconnect = async (serverUrl: string) => {
-    const database = DatabaseManager.serverDatabases[serverUrl];
-    if (!database) {
-        return {error: `${serverUrl} database not found`};
+export const updateAllUsersSince = async (serverUrl: string, since: number, fetchOnly = false) => {
+    if (!since) {
```

Comment:
Just to make sure. In this case, "updateAllUsersSince" 0, does not mean update from the beginning, but don't update at all. Is it right?

Reply:
exactly right